doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -218,6 +218,8 @@ nRF Desktop
 
   * Removed separate configurations enabling :ref:`zephyr:shell_api` (:file:`prj_shell.conf`).
     Shell support can be enabled for a given configuration with the single Kconfig option (:ref:`CONFIG_DESKTOP_SHELL <config_desktop_app_options>`).
+  * nRF Desktop dongles by default use Bluetooth appearance (:kconfig:option:`CONFIG_BT_DEVICE_APPEARANCE)` of Keyboard.
+    This improves consistency with the used HID boot interface.
```

Comment:
Would it be correct to say "this setting improves (...)" ?

Reply:
Maybe `The new default configuration value improves ... `?

Reply:
Yup, sounds great!